R/mod_baseline_adjustment_server.R

```diff
@@ -41,13 +41,13 @@ mod_baseline_adjustment_server <- function(id, params) {
       dplyr::mutate(id = glue::glue("{at}_{g}_{sanitized_code}"))
 
     # reactives ----
-    baseline_data <- shiny::reactive({
+    baseline_data_filtered <- shiny::reactive({
       # nolint start: object_usage_linter
       dataset <- shiny::req(params$dataset)
       year <- as.character(shiny::req(params$start_year))
       # nolint end
 
-      load_provider_data("baseline") |>
+      baseline_data |>
```

Comment:
```suggestion
      baseline_data |>
```

Does this not need `()` as it's reactive? Or is that not required when we're using within a reactive?

This occurs a few placed in this PR - I've only noted it here incase I'm being stupid

Reply:
For some reason we are passing the evaluated reactives to the modules. Might have been to make it easier to test the modules? but could do with being looked at in the future